adoc/Release-Notes.adoc

```diff
@@ -33,6 +33,67 @@ endif::[]
 [id='sec.major-change']
 == Major Changes
 
+[id='sec.1_5']
+=== Release 1.5, September 2019
+
+[id='sec.1_5.new']
+==== What Is New?
+
+* {scf} has been updated to version 2.18.0.
+
+[id='sec.1_5.issue']
+==== Known Issues
+
+IMPORTANT: In order to deploy to this version, or upgrade from {scf} 2.17.1, part of {cap} 1.4.1, to this version, you will need to scale down the database roles to 1 pod in order to migrate from cf-mysql to PXC. This is based on upstream instructions but based on what we've seen with other components that rely on the database, such as Credhub, scaling all database roles into single availability helps with a stable migration and deployment
+
+Steps to upgrade: [verified by Docs, need to format properly for AsciiDocs]
+
+========== Install 1.4.1, full HA ==========
+UAA:
+helm install --name scf-uaa --namespace uaa suse/uaa -f <values.yaml> --set config.HA=true
+
+SCF:
+helm install --name scf-cf --namespace scf suse/cf -f <values.yaml> --set config.HA=true --set "secrets.UAA_CA_CERT=${CA_CERT}"
+
+========== Upgrade 1.4.1, HA but scale mysql to 1 ==========
+UAA:
+helm upgrade scf-uaa --namespace uaa suse/uaa --reuse-values --set config.HA=false --set sizing.uaa.count=2 --set sizing.mysql.count=1
+
+SCF:
+helm upgrade scf-cf --namespace scf suse/cf --reuse-values --set config.HA=false --set=sizing.diego_cell.count=3 --set=sizing.adapter.count=2 --set=sizing.api_group.count=2 --set=sizing.autoscaler_actors.count=2 --set=sizing.autoscaler_api.count=2 --set=sizing.autoscaler_metrics.count=2 --set=sizing.cc_clock.count=2 --set=sizing.cc_uploader.count=2 --set=sizing.cc_worker.count=2 --set=sizing.cf_usb_group.count=2 --set=sizing.diego_api.count=2 --set=sizing.diego_brain.count=2 --set=sizing.diego_ssh.count=2 --set=sizing.doppler.count=2 --set=sizing.locket.count=2 --set=sizing.log_api.count=2 --set=sizing.log_cache_scheduler.count=2 --set=sizing.nats.count=2 --set=sizing.nfs_broker.count=2 --set=sizing.router.count=2 --set=sizing.routing_api.count=2 --set=sizing.syslog_scheduler.count=2 --set=sizing.tcp_router.count=2 --set=sizing.mysql.count=1
+
+Delete the PVC(s) for mysql-1 from both uaa and scf namespaces.
+
+========== Upgrade to 1.5RC, HA but mysql at 1 ==========
+UAA:
+helm upgrade scf-uaa --namespace uaa <chart-path>  -f <rc-values.yaml> --set config.HA=true --set config.HA_strict=false --set sizing.mysql.count=1 --set sizing.uaa.count=2
+
+SCF:
+helm upgrade scf-cf --namespace scf <chart-path>  -f <rc-values.yaml> --set config.HA=true --set config.HA_strict=false --set sizing.mysql.count=1 --set "secrets.UAA_CA_CERT=${CA_CERT}"
+
+========== Upgrade 1.5RC, full HA ==========
+
+UAA:
+helm upgrade scf-uaa --namespace uaa <chart-path> -f <rc-values.yaml> --set config.HA_strict=true --set config.HA=true
```

Comment:
I think config.HA_strict i already true by default. Since we are not using ----reuse-values, we should need `config.HA_strict=true`

Reply:
I'm OK leaving that parameter in just so admins are aware of this parameter in general.